static/policies/privacy.html

```diff
@@ -39,7 +39,11 @@ <h3>Your data</h3>
     window along with your chosen compiler and options to the Compiler Explorer servers. There, the source code is
     written to disk and your chosen compiler is invoked on it. If your request was to have your code executed, the
     resulting executable is run. The outputs from compilation and execution are processed and sent back to your web
-    browser, where they're shown. Once this process completed, your source code is deleted from disk.
+    browser, where they're shown. Shortly after this process completes, your source code is deleted from disk. If, in
+    processing your query, an issue with Compiler Explorer is found, your code may be kept a little longer in order to
```

Comment:
I think instead of being a bit vague about how long it can be kept and then specifiec in the next line(s), we could move the one week stuff here, so something like this perhaps:
```diff
- your code may be kept a little longer in order to
+ your code may be kept for up to a week in order to
```
Or maybe as done on the other chunk with the parenthesis?

Reply:
Yeah that's better. I kept going back and forth on specifying the length